rfqmsg/buy_accept.go

```diff
@@ -44,15 +42,14 @@ type BuyAccept struct {
 //
 // TODO(ffranr): Use new AssetRate type for assetRate arg.
 func NewBuyAcceptFromRequest(request BuyRequest,
-	assetRate rfqmath.BigIntFixedPoint, expiry uint64) *BuyAccept {
+	assetRate rfqmath.BigIntFixedPoint, expiry time.Time) *BuyAccept {
```

Comment:
nit: feels like we could also fix the TODO above with this commit? Since we now have the correct data types in place.